cartoframes/viz/__init__.py

```diff
@@ -17,7 +17,6 @@
     'Source',
     'Style',
     'Popup',
-    'Kuviz',
     'KuvizPublisher',
```

Comment:
Does the `KuvizPublisher` need to be exposed to the user? I don't see it used in the example notebook

Reply:
+1

Reply:
:+1: